src/App.tsx

```diff
@@ -17,13 +25,45 @@ function App() {
     };
 
     setTaskList([...taskList, newTask]);
+
+    setTaskText('');
+  };
+
+  const clickEdit = (id: string) => {
+    const taskToEdit: Task[] = taskList.filter((task: Task) => {
+      return task.id === id;
+    });
+
+    setSelectedTask(taskToEdit[0]);
+
+    setTaskText(taskToEdit[0].textTask);
```

Comment:
Este espaço entre as funções não é necessários
Geralmente você quer separar em blocos mesmo como você já faz, mas acho que isto aqui poderia estar junto, em todas as functions
```suggestion
    setSelectedTask(taskToEdit[0]);
    setTaskText(taskToEdit[0].textTask);
```